worker/contract_lock.go

```diff
@@ -26,9 +26,9 @@ type contractLock struct {
 	locker ContractLocker
 	logger *zap.SugaredLogger
 
-	stopCtx       context.Context
-	stopCtxCancel context.CancelFunc
-	stopWG        sync.WaitGroup
+	keepaliveLoopCtx       context.Context
```

Comment:
Was there a reason why we are not just calling all those `shutdownCtx` and `keepaliveLoopCtx` `ctx`? We can just make it a convention that a type can have a `ctx` which indicates when it's stopped/closed.

Reply:
I'm not against that. I kind of liked `shutdownCtx` convention because it adds a bit more context but fine to just rename it to `ctx`. I renamed this `keepaliveLoopCtx` for the same reason.

Reply:
I'll resolve this but fine to rename, I'll ping you in DM.